server/models/Group.js

```diff
@@ -382,8 +382,12 @@ module.exports = function(Sequelize, DataTypes) {
       },
 
       getRelatedGroups(limit) {
```

Comment:
I like that, but I'd rename it `settings.includeCollectivesByTag` (or withTags)
(so that a new reader doesn't have to understand the concept of super collective to understand what this does)